static/app/components/replays/breadcrumbs/replayComparisonModal.tsx

```diff
@@ -42,6 +44,12 @@ export default function ReplayComparisonModal({
 
   const [leftBody, setLeftBody] = useState(null);
   const [rightBody, setRightBody] = useState(null);
+  let startOffset = leftTimestamp - 1;
+  // If the error occurs close to the start of the replay, clamp the start offset to 1
+  // to help compare with the html provided by the server, This helps with some errors on localhost.
+  if (startOffset < MAX_CLAMP_TO_START) {
+    startOffset = 1;
+  }
```

Comment:
We could also look at the url if you want and just enable this for replays in local host. 

I'm slightly worried that, since it's a timing thing, 2,000 won't be reliable, some sites could load faster or slower. 

We could also do this and/or build a little "adjust frame offset" button too (I'm thinking along the lines of the replay preview snippet, that might have the same kind of frame-by-frame stepping)

Reply:
i was seeing that it worked the same on the examples i tried, lets try it out and i'll take a look at some more examples. The production ones were usually even closer to 0. usually 100-200ms offset